media-libs/vulkan-loader/vulkan-loader-1.2.137.ebuild

```diff
@@ -49,7 +49,7 @@ multilib_src_configure() {
 		-DBUILD_WSI_WAYLAND_SUPPORT=$(usex wayland)
```

Comment:
1.2.137 is stable so please copy it do a new ebuild named 1.2.137-r1, drop amd64 and x86 to testing, and make your change

Reply:
Same for 1.2.141, sorry